docs/utilities/parameters.md

```diff
@@ -334,94 +313,50 @@ DynamoDB provider can be customized at initialization to match your table struct
 | **sort_attr**  | No        | `sk`    | Range key for the DynamoDB table. You don't need to set this if you don't use the `get_multiple()` method. |
 | **value_attr** | No        | `value` | Name of the attribute containing the parameter value.                                                      |
 
-```python hl_lines="3-8" title="Customizing DynamoDBProvider to suit your table design"
-from aws_lambda_powertools.utilities import parameters
-
-dynamodb_provider = parameters.DynamoDBProvider(
-	table_name="my-table",
-	key_attr="MyKeyAttr",
-	sort_attr="MySortAttr",
-	value_attr="MyvalueAttr"
-)
+=== "builtin_provider_dynamodb_custom_fields.py"
+    ```python hl_lines="3 5 9 10 15"
```

Comment:
On the rendered docs the highlight seems wrong. Can you please review it?

Reply:
Yep! Fixed.